pkg/gitreceive/build.go

```diff
@@ -178,14 +190,19 @@ func build(
 		)
 	} else {
 		buildPodName = slugBuilderPodName(appName, gitSha.Short())
+
+		cacheKey := ""
+		if !slugBuilderInfo.DisableCaching() {
+			cacheKey = slugBuilderInfo.CacheKey()
+		}
```

Comment:
Why did you add this exactly? When the cache is disabled, the cache key should still be a correct 'cache key', right? It should just be disabled, which it is.